WordPress/Classes/ViewRelated/Ratings/AppFeedbackPromptView.swift

```diff
@@ -77,20 +78,23 @@ class AppFeedbackPromptView: UIView {
         leftButton.removeTarget(nil, action: nil, for: .touchUpInside)
         leftButton.setTitle(title, for: .normal)
         leftButton.on(.touchUpInside, call: tapHandler)
+        evaluateStackAxisMode()
     }
 
     func setupNoButton(title: String, tapHandler: @escaping (UIControl) -> Void) {
         rightButton.removeTarget(nil, action: nil, for: .touchUpInside)
         rightButton.setTitle(title, for: .normal)
         rightButton.on(.touchUpInside, call: tapHandler)
+        evaluateStackAxisMode()
     }
 
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
+        evaluateStackAxisMode()
+    }
 
-        buttonStack.axis = .horizontal
-        buttonStack.isLayoutMarginsRelativeArrangement = true
-
+    /// Evaluate the width of the buttons to determine if the stack view should go into vertical mode.
+    private func evaluateStackAxisMode() {
```

Comment:
This is now being called _after_ the text is set so it can factor in the true button size and determine if it should switch the stack view to vertical mode.